helpers/config.py

```diff
@@ -326,6 +324,12 @@ def get_config_template(cls):
             "mongo_user_username": "kobo",
             "mongo_user_password": Config.generate_password(),
             "redis_password": Config.generate_password(),
+            "uwsgi_workers_start": "1",
+            "uwsgi_workers_max": "2",
+            "uwsgi_max_requests": "512",
+            "uwsgi_soft_limit": "128",
+            "uwsgi_haraki": "120",
```

Comment:
`harakiri` (two syllables): https://en.wikipedia.org/wiki/Seppuku
(probably not the best thing to casually reference in a web server configuration :facepalm:)